src/components/dialogs/create-study-dialog.js

```diff
@@ -220,6 +220,13 @@ export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
             });
     };
 
+    useEffect(() => {
+        if (!open && tempCaseUuid !== null) {
+            deleteCase(tempCaseUuid).catch((error) => console.error(error));
+            setTempCaseUuid(null);
```

Comment:
deleteCase(tempCaseUuid).then((r) => setTempCaseUuid(null)).catch((error) => console.error(error));

Reply:
i changed it